weblate/trans/tests/test_reports.py

```diff
@@ -127,12 +127,7 @@ def get_credits(self, style):
         self.add_change()
         return self.client.post(
             reverse("credits", kwargs=self.get_kwargs()),
-            {
-                "period": "",
-                "style": style,
-                "start_date": "2000-01-01",
-                "end_date": "2100-01-01",
-            },
+            {"style": style, "period": "01/01/2000 - 01/01/2100"},
```

Comment:
Please add tests for invalid period input.

Reply:
Tests were added including a test for a blank range value.